views/base-head.html.twig

```diff
@@ -47,4 +47,11 @@
     gtag('js', new Date());
     gtag('config', {{ app.getConfig('google_analytics_property')|json_encode|raw }});
 </script>
+{% else %}
+<!-- Analytics Tracking Disabled -->
+<script>
+var ga = function () {
+  console.log('Google Analytics Mock:', arguments);
```

Comment:
TIL I can probably stop worrying about browsers that don't support `console.log`
https://developer.mozilla.org/en-US/docs/Web/API/Console/log#Browser_compatibility

Reply:
Yeah, in theory we wouldn't deploy HealthPro without the tracking code set so I didn't worry about it.